internal/hud/server/server_test.go

```diff
@@ -325,31 +325,38 @@ func TestMaybeSendToTriggerQueue(t *testing.T) {
 	assert.Equal(t, "foobar", action.Name.String())
 }
 
-func TestMaybeSendToTriggerQueue_noManifestWithName(t *testing.T) {
-	f := newTestFixture(t)
-
-	err := server.MaybeSendToTriggerQueue(f.st, "foobar")
-
-	assert.EqualError(t, err, "no manifest found with name 'foobar'")
-	store.AssertNoActionOfType(t, reflect.TypeOf(server.AppendToTriggerQueueAction{}), f.getActions)
-}
-
-func TestMaybeSendToTriggerQueue_notManualManifest(t *testing.T) {
+func TestSendToTriggerQueue_automaticManifest(t *testing.T) {
 	f := newTestFixture(t)
 
 	mt := store.ManifestTarget{
 		Manifest: model.Manifest{
 			Name:        "foobar",
-			TriggerMode: model.TriggerModeAuto,
+			TriggerMode: model.TriggerModeAuto, // can now enqueue manifests regardless of TriggerMode
```

Comment:
Comments with "non" get outdated quickly. Not sure if this is relevant. If anything, "also test auto" or some such?

Reply:
fair, i'll just remove it